.github/workflows/cd.yml

```diff
@@ -173,6 +173,10 @@ jobs:
         run: |
           git add packages/fx-core/src/component/m365/serviceConstant.ts
           git commit -m "build: replace sideloading placeholders"
+      
+      - name: enable prerelease only features
+        if: ${{ github.event_name == 'workflow_dispatch' && (github.event.inputs.preid == 'alpha' || github.event.inputs.preid == 'beta') }}
```

Comment:
missed the scheduled event
${{ github.event == schedule || (github.event_name == 'workflow_dispatch' ……) }}

Reply:
Thanks! Updated